tutorial.md

```diff
@@ -1,17 +1,24 @@
 # Boson Functions: A Step By Step Tutorial
+
 This document will walk you step by step through the process of creating,
 editing, and deploying a Boson Function project.
 
 ## Prerequisites
-In order to follow along with this tutorial, you will need to have a few
-tools installed. This project currently is limited to Linux.
 
-* [Docker](https://docs.docker.com/install/)
-* [oc](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands) or [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) CLI
-* [kn](https://knative.dev/docs/install/install-kn/) CLI
+In order to follow along with this tutorial, you will need to have a few tools
+installed.
+
+* [oc][oc] or [kubectl][kubectl] CLI
+* [kn][kn] CLI
+* [Docker][docker] 
 
+[docker]: https://docs.docker.com/install/
+[oc]: https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands
```

Comment:
just a nit, we might want to target OCP 4.6 in the links (I doubt anything was changed there, but you know)